crash-reporting/shopify.app.raygun-crash-reporting.toml

```diff
@@ -0,0 +1,19 @@
+# Learn more about configuring your app at https://shopify.dev/docs/apps/tools/cli/configuration
+
+name = "Raygun Crash Reporting"
+client_id = "242392a59ff87b609c4027eee44772c7"
+application_url = "https://shopify.dev/apps/default-app-home"
+embedded = false
```

Comment:
`embedded = false` is in here twice?

Reply:
The first embedded is for an embedded app and the second is for a [POS](https://shopify.dev/docs/apps/pos/getting-started) app. I believe I can remove the POS one as it's not required